### PR TITLE
Add WE addon installation test

### PR DESCRIPTION
**Pull request by mitiao**
Confirmed with Frederic that no release note for WE.

### CODE REVIEW

File: tests/installation/releasenotes.pm [hunk 1]
```diff
@@ -28,6 +28,7 @@ sub run() {
     }
     if (@addons) {
         for $a (@addons) {
+            next if ($a =~ /we/i);                     # https://bugzilla.suse.com/show_bug.cgi?id=931003#c17
```
Comment (dzedro) [inline tests/installation/releasenotes.pm]: too much "i" ? :)

Comment (mitiao) [inline tests/installation/releasenotes.pm]: oh, yes.
SCC_ADDONS also should be lower case :) 
will remove it then

**Comment by aaannz** [comment]
Can you then please squash commits to one? It's unnecessary to have 3 commits for one line.

**Comment by mitiao** [comment]
Sure.

**Comment by aaannz** [comment]
Well, you don't need to close this PR. Once you squash them (`git rebase -i` is you friend) you can force push it to the same branch (`git push -f`)

**Pull request closed without merging**